cluster/examples/cache/rediscluster/resource-claim.yaml

```diff
@@ -3,10 +3,7 @@ apiVersion: cache.crossplane.io/v1alpha1
 kind: RedisCluster
 metadata:
   name: app-redis
-  namespace: app-project1-dev
```

Comment:
Noticed the `namespace` is removed on all of the claims. Is this just for simplicity?

Reply:
Exactly. I figured given they were examples it was fine for everything to be created in `default` or whatever namespace was passed to `kubectl -n`.